tests/test/scheduler/test_mpi_world.cpp

```diff
@@ -112,48 +112,94 @@ TEST_CASE("Test cartesian communicator", "[mpi]")
     cleanFaabric();
 
     faabric::Message msg = faabric::util::messageFactory(user, func);
-    // 5 processes create a 2x2 grid with one left MPI_UNDEFINED
```

Comment:
This was false in LAMMPS. In particular, my wrong assumption was that MPI inferred the size of the grid given the `worldSize`. However, I realized that the programmer provides it.